website/content/docs/nia/index.mdx

```diff
@@ -32,6 +32,15 @@ Consul-Terraform-Sync executes one or more automation tasks with the most recent
 
 **Terraform Provider** - A [Terraform provider](https://www.terraform.io/docs/providers/index.html) is responsible for understanding API interactions and exposing resources for an infrastructure type.
 
+## Getting Started With Network Infrastructure Automation
+
+The [Network Infrastructure Automation (NIA)](https://learn.hashicorp.com/collections/consul/network-infrastructure-automationutm_source=WEBSITE&utm_medium=WEB_IO&utm_offer=ARTICLE_PAGE&utm_content=DOCS) 
```

Comment:
I like the threading of Learn links here. Though this one doesn't seem to work and I get Page Not Found when clicking from the vercel preview app

![image](https://user-images.githubusercontent.com/6362111/111534542-1d085080-8736-11eb-9142-e7146138ae18.png)

Reply:
Oh i think it's just missing a `?`

```suggestion
The [Network Infrastructure Automation (NIA)](https://learn.hashicorp.com/collections/consul/network-infrastructure-automation?utm_source=WEBSITE&utm_medium=WEB_IO&utm_offer=ARTICLE_PAGE&utm_content=DOCS) 
```